web/src/main/java/it/pagopa/selfcare/onboarding/web/model/mapper/OnboardingInstitutionInfoMapper.java

```diff
@@ -35,18 +35,18 @@ default List<GeographicTaxonomyResource> mapGeographicTaxonomies(List<Geographic
                 .orElse(null);
     }
 
-
-    default InstitutionData toDataImpl(InstitutionInfo model) {
-        return toData(model);
+    @Named("toDataImpl")
```

Comment:
it might be better to give more meaning rather "toDataImpl"....for ex. "toInstitutionData"